rotkehlchen/tests/unit/decoders/test_uniswapv4.py

```diff
@@ -330,3 +339,249 @@ def test_multi_pool_swap_to_second_address(
+        location=Location.ETHEREUM,
+        event_type=HistoryEventType.BURN,
+        event_subtype=HistoryEventSubType.NFT,
+        asset=Asset('eip155:1/erc721:0xbD216513d74C8cf14cf4747E6AaA6420FF64ee9e/51028'),
+        amount=ONE,
+        location_label=user_address,
+        notes='Exit Uniswap V4 LP with id 51028',
+        counterparty=CPT_UNISWAP_V4,
+        address=ZERO_ADDRESS,
+    ), EvmEvent(
+        tx_hash=tx_hash,
+        sequence_index=2,
+        timestamp=timestamp,
+        location=Location.ETHEREUM,
+        event_type=HistoryEventType.WITHDRAWAL,
+        event_subtype=HistoryEventSubType.REDEEM_WRAPPED,
+        asset=A_ETH,
+        amount=FVal(eth_amount := '0.01122274372981923'),
+        location_label=user_address,
+        notes=f'Withdraw {eth_amount} ETH from Uniswap V4 LP',
+        counterparty=CPT_UNISWAP_V4,
+        address=(pool_manager := string_to_evm_address('0x000000000004444c5dc75cB358380D2e3dE08A90')),  # noqa: E501
+    ), EvmEvent(
+        tx_hash=tx_hash,
+        sequence_index=3,
+        timestamp=timestamp,
+        location=Location.ETHEREUM,
+        event_type=HistoryEventType.WITHDRAWAL,
+        event_subtype=HistoryEventSubType.REDEEM_WRAPPED,
+        asset=A_USDC,
+        amount=FVal(usdc_amount := '175118.699175'),
+        location_label=user_address,
+        notes=f'Withdraw {usdc_amount} USDC from Uniswap V4 LP',
+        counterparty=CPT_UNISWAP_V4,
+        address=pool_manager,
+    )]
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('polygon_pos_accounts', [['0x2fCC69eaa8c9F33538CBac50eb5432c422825e6D']])
+def test_decrease_liquidity(
+        polygon_pos_inquirer: 'PolygonPOSInquirer',
+        polygon_pos_accounts: list['ChecksumEvmAddress'],
+) -> None:
+    tx_hash = deserialize_evm_tx_hash('0x550832040e57647103a03e23eba5ce869f8a96c3e8b92f1f9258024a286e2525')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(evm_inquirer=polygon_pos_inquirer, tx_hash=tx_hash)  # noqa: E501
+    assert events == [EvmEvent(
+        tx_hash=tx_hash,
+        sequence_index=0,
+        timestamp=(timestamp := TimestampMS(1755609920000)),
+        location=Location.POLYGON_POS,
+        event_type=HistoryEventType.SPEND,
+        event_subtype=HistoryEventSubType.FEE,
+        asset=A_POLYGON_POS_MATIC,
+        amount=FVal('0.005868403632678156'),
+        location_label=(user_address := polygon_pos_accounts[0]),
+        counterparty=CPT_GAS,
+        notes='Burn 0.005868403632678156 POL for gas',
+    ), EvmEvent(
+        tx_hash=tx_hash,
+        sequence_index=1,
+        timestamp=timestamp,
+        location=Location.POLYGON_POS,
+        event_type=HistoryEventType.WITHDRAWAL,
+        event_subtype=HistoryEventSubType.REDEEM_WRAPPED,
+        asset=A_POLYGON_POS_MATIC,
+        amount=FVal('20.49016382840257149'),
+        location_label=user_address,
+        notes='Withdraw 20.49016382840257149 POL from Uniswap V4 LP',
```

Comment:
why not include the nft id here in the extra data or in the notes? i think that'd be helpful.